.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master, jf/ops ]
+  pull_request:
+    branches: [ master, jf/ops ]
```

Comment:
jf/ops is here for testing, we can remove just before merge or afterwards in a clean up PR